src/content/en/updates/2020/08/devtools.md

```diff
@@ -0,0 +1,261 @@
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/660b9dfbe395bb6e230ca639fce11afa025d9ae3 #}
+
+Chromium issues: [1086817](https://crbug.com/1086817), [1080569](https://crbug.com/1080569)
+
+
+## Lighthouse 6.2 in the Lighthouse panel {: #lighthouse }
+The Lighthouse panel is now running Lighthouse 6.2. Lighthouse will now enhance the unused-javascript audit if a page has publicly-accessible JavaScript source maps. Check out the [release notes](https://github.com/GoogleChrome/lighthouse/releases/tag/v6.2.0) for a full list of all changes.
+
+![Unsize image](/web/updates/images/2020/08/lighthouse-unsize-img.png)
+
+These are the new audits added in Lighthouse:
+
+- **Avoid long main thread tasks**. Reports the longest tasks on the main thread, useful for identifying worst contributors to input delay.
+- **Links are crawlable**. Check if the `href` attribute of anchor elements links to an appropriate destination, so the links can be discovered.
+- **Unsized image elements** - Check if an explicit `width` and `height` is set on image elements. Explicit image size can reduce layout shifts and improve CLS.
+- **Avoid non-composited animations**. Reports [non-composited animations](https://web.dev/non-composited-animations/) that appear janky and reduce CLS.
+- **Listens for the `unload` events**. Reports the `unload` event. Consider using the `pagehide` or `visibilitychange` events instead as the `unload` event does not fire reliably.
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend.git/+/c55849a7826fcabbae137b0f85cd44c93498581d #}
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/ca50c1ef5f527bd1e8dd86b53278c7da62d25681 #}
+
+Chromium issue: [772558](https://crbug.com/772558)
+
+
+## Deprecation of “other origins” listing in the Service Workers pane {: #deprecate-sw-other-origins }
+
+DevTools now provides a link to view the full list of service workers from other origins in a new browser tab - [chrome://serviceworker-internals/?devtools](chrome://serviceworker-internals/?devtools).
+
+Previously DevTools displayed a list nested under the Application panel > Service workers.
+
+![Link to other origins](/web/updates/images/2020/08/sw-other-origins.png)
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/fbf20a6507edffeab481efedd1e8de710e4cbfbe #}
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/bb22c1539f01c02985b0b116cebde2143c0e1b75  #}
+
+Chromium issue: [807440](https://crbug.com/807440)
+
+
+## Show coverage summary for filtered items {: #filter-coverage-summary }
+DevTools now recalculates and displays a summary of coverage information dynamically when filters are applied in the Coverage tab. Previously, the Coverage tab always displays a summary of all coverage information. 
+
+The following example shows the coverage summary when no filter applied versus after applying filter by CSS type. 
```

Comment:
```suggestion
In the example below notice how the summary initially says
`446 kB of 2.0 MB (22%) used so far. 1.5 MB unused.` and then says
`57 kB of 604 kB (10%) used so far. 546 kB unused.` after CSS filtering has been applied.
```

I want to call more attention to the new thing, because when I first read this I was confused because I thought you were saying that CSS/JS filtering was new (it's not)